src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java

```diff
@@ -24,12 +24,14 @@
      * Type B is a class that extends {@link AbstractBuildingWorker.View}.
      */
     protected final B          building;
-    private final   SwitchView switchView;
     private final   Label      title;
-    private final   Button     buttonPrevPage;
-    private final   Button     buttonNextPage;
     private final   Button     buttonBuild;
     private final   Button     buttonRepair;
+    protected       SwitchView switchView;
+    protected       int        switchPagesSize;
+    protected final Label      pageNum;
+    protected final Button     buttonPrevPage;
```

Comment:
### Errors:
- Protected field buttonPrevPage 
- Type of field 'buttonPrevPage' is concrete class Button 
- Can be private

### Warnings:
- Instance field name buttonPrevPage doesn't match regex 'm_[a-z][A-Za-z\d]*'